commands/output/output.go

```diff
@@ -170,6 +178,7 @@ func PrintDetails(r *review.Review) error {
 	fmt.Printf(reviewDetailsTemplate, r.Request.ReviewRef, r.Request.TargetRef,
 		strings.Join(r.Request.Reviewers, ", "),
 		r.Request.Requester, r.GetBuildStatusMessage())
+	printBuildStatusOutput(r)
```

Comment:
This should probably not be displayed by default (that would be very verbose), but rather limited to the case of a user passing in a special flag.

Specifically, I mean something like what we are doing for displaying a diff only if the `--diff` flag is passed in.